sound/soc/sof/core.c

```diff
@@ -16,6 +16,9 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+bool no_PM_support;
+EXPORT_SYMBOL(no_PM_support);
+
```

Comment:
@ranj063 I'm not a fan of exporting global variables. Maybe we can integrate this inside sof_dev as another op.
Another option would be to prevent entering runtime suspend if runtime_suspend is not present.

Reply:
@dbaluta agree with this. I'll need to think about a more elegant way to do this.